Readme (#17)
Azure/terraform-azurerm-naming: README.md

```tf
module "naming" "main" {
  source  = "Azure/naming/azurerm"
  suffix = [ "test" ]
}
resource "azurerm_resource_group" "example" {
  name     = module.naming.main.resource_group.name
  location = "West Europe"
}
```

> Other advances usages will be explained in the [Advanced usage](#advancedusage) part of this docs.

Reviewer: advances -> advanced

## Modifying resources

> The resources are automatically generated using `go` to change the generation please change the file on the `templates` folder. To add a new resource, including its definition in the file `resourceDefinition.json`, and it will be automatically generated when `main.go` is run.

Reviewer: Might need rewording a little, hard to follow.

# Current implementation

You can find a list bellow of all the resources that are currently implemented. To get a list of the ones that are missing implementation you can check at [Missing resources](docs/missing_ressources.md) the resources that have no documentation about their limitation on naming currently on Microsoft docs are on the [Not defined](docs/not_defined.md) list.

# Advanced usage

## Output

> Each one of the resources spits out more than just the name of the resource but also other properties:

Reviewer 2: ```suggestion
Each one of the resources emits the name of the resource and other properties:
```

| Property | Type | Description |
| ----- |----- | ---- |

> | name | string | name of the resource including their respective suffixes and prefixes applied |

Reviewer 2: ```suggestion
| name | string | name of the resource including respective suffixes and prefixes applied |
```
